.travis.yml

```diff
@@ -40,21 +40,21 @@ jobs:
         - curl --data '{"tag_name":"'$SERVER_RELEASE_NAME'","name":"'$SERVER_RELEASE_NAME'","prerelease":true}' $CREATE_RELEASE_URL
         - MANAGER_RELEASE_NAME=v$(date -I)
         - curl --data '{"tag_name":"'$MANAGER_RELEASE_NAME'","name":"'$MANAGER_RELEASE_NAME'","prerelease":true}' $CREATE_RELEASE_URL
-    - stage: "Deploy Server"	
-      name: Server Testing	
-      sudo: required	
-      services: docker	
-      script:	
-        # https://docs.travis-ci.com/user/docker/	
-        - |	
-          sudo rm -f /usr/local/bin/docker-compose	
-          curl -L https://github.com/docker/compose/releases/download/1.25.5/docker-compose-$(uname -s)-$(uname -m) > docker-compose	
-          chmod +x docker-compose	
-          sudo mv docker-compose /usr/local/bin	
+    - stage: "Deploy Server"
+      name: Server Testing
+      sudo: required
+      services: docker
+      script:
+        # https://docs.travis-ci.com/user/docker/
+        - |
+          sudo rm -f /usr/local/bin/docker-compose
+          curl -L https://github.com/docker/compose/releases/download/1.25.5/docker-compose-$(uname -s)-$(uname -m) > docker-compose
+          chmod +x docker-compose
+          sudo mv docker-compose /usr/local/bin
         - yarn do shadowbox/test
         - yarn do shadowbox/docker/build && cd src/shadowbox/integration_test && ./test.sh
-    
-    
+
+
```

Comment:
nit: extra newline

Reply:
Done in 615c0c702c41fd02d8c9ae4d37dc72560976e7e8